change/@fluentui-web-components-2021-01-18-14-20-53-bump-webpack-webcomponents.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "patch",
```

Comment:
As this does not change distributed files, the package version shouldn’t update. Can you make this “none”?

Reply:
Fixed!